### PR TITLE
Completed css-two-column-layout exercise.-Lisa Le

### PR DESCRIPTION
**Pull request by lisale841**
5837A36D-A585-4581-9551-11397811F581

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+              We've dedicated thousands of man hours to making healthcare
+              instituions in the Helium network run 10x more efficiently and
+              deliver an unparalleled quality of care.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-two">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: `column-two`? Is this meant to be something like `column-half`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! One note for you:

File: css-two-column-layout/index.html (hunk 1 above)
Comment (lisale841) [reply css-two-column-layout/index.html]: Hi Cody.  Thank you.  Yes i meant column-half.  I just updated and push the update with column-half. :)

**Review by codingcodymiller — APPROVED** [review]
Excellent work on this!!

**Pull request merged**